packages/asana/test/adaptor.test.js

```diff
@@ -241,5 +242,40 @@ describe('Adaptor Test', () => {
       expect(error).to.be.an('Error');
       expect(error.message).to.equal('You need to specify Workspace GID');
     });
+    it('should query a task with custom fields', async () => {
```

Comment:
So this test just asserts that we can already use custom fields? We don't  need any extra logic to support that feature?

I would like to see these tests referenced in the PR description somewhere, because they're presently unrelated to the PR (and so untraceable and out of scope)

Reply:
@josephjclark I have updated the PR notes to cover the unit test that i have updated